src/ProductTour/Checkpoint.scss

```diff
@@ -0,0 +1,130 @@
+@import "variables";
+
+$checkpoint-arrow-top-color: solid $checkpoint-arrow-width $checkpoint-border-color;
+$checkpoint-arrow-default-color: solid $checkpoint-arrow-width $checkpoint-background-color;
+$checkpoint-arrow-transparent: solid $checkpoint-arrow-width transparent;
+
+.pgn__checkpoint {
+  position: absolute;
+  background: $checkpoint-background-color;
+  border-top: $checkpoint-border-width solid $checkpoint-border-color;
+  border-radius: $border-radius;
+  z-index: $checkpoint-z-index;
+  max-width: $checkpoint-max-width;
+  @media (max-width: map-get($grid-breakpoints, 'md')) {
+    min-width: 90%;
+    max-width: 90%;
+  }
+
+  #pgn__checkpoint-arrow,
+  #pgn__checkpoint-arrow::before,
+  #pgn__checkpoint-arrow::after {
+    position: absolute;
+    width: 0;
+    height: 0;
+  }
+
+  #pgn__checkpoint-arrow {
+    visibility: hidden;
+  }
+
+  #pgn__checkpoint-arrow::before,
+  #pgn__checkpoint-arrow::after {
+    visibility: visible;
+    content: '';
+  }
+
+  .pgn__checkpoint-button_dismiss {
+    margin-right: map_get($spacers, 2);
+  }
+
+  .pgn__checkpoint-breadcrumb {
+    height: 6px;
+    width: 6px;
+    border-radius: 50%;
+
+    &.pgn__checkpoint-breadcrumb_active {
+      background: $checkpoint-breadcrumb-color;
+    }
+
+    &.pgn__checkpoint-breadcrumb_inactive {
+      border: 1px solid $checkpoint-breadcrumb-color;
+      background: transparent;
+    }
+  }
+
+  .pgn__checkpoint-breadcrumb:not(:first-child) {
+    margin-left: map_get($spacers, 2);
+  }
+
+  .pgn__checkpoint-breadcrumb-container {
+    display: flex;
+    align-items: center;
+  }
+
+  .pgn__checkpoint-body {
+    color: $checkpoint-body-color;
+    margin-bottom: map_get($spacers, 4);
+  }
+
+  #pgn__checkpoint-title {
+    font-size: $h3-font-size;
+    margin-right: map_get($spacers, 3);
```

Comment:
TODO: $spacers adjustment
breadcrumb, 1.5
body, 3.5
title, 2.5

It didn't seem like the decimal places were working in the spacers, but maybe it just wasn't rendering for me correctly locally

Reply:
Fixed this by using `1\.5` instead of `1.5`, for example. That's what the definition for the spacers map did. I am not 100% but I think that's escaping the period so it's parsed as a string, not a number. Seemed to work for me anyway.

So I made your suggested tightening of spacers.